laghos_rom.cpp

```diff
@@ -375,9 +375,28 @@ ROM_Basis::ROM_Basis(ROM_Options const& input, Vector const& S, MPI_Comm comm_)
         initV = new CAROM::Vector(tH1size, true);
         initE = new CAROM::Vector(tL2size, true);
 
-        //TODO: Tony param init after PR 77
-        if (input.paramOffset)
+        // std::string path_init = (parameterID >= 0) ? "run/ROMoffset/param" + std::to_string(parameterID) + "_init" : "run/ROMoffset/init"; // TODO: Tony PR77
+        std::string path_init = "run/ROMoffset/init";
```

Comment:
I pushed some changes to rom-dev that will affect this line. Can you change it to 

std::string path_init = basename + "/ROMoffset/init";